readme.md

```diff
@@ -135,6 +135,13 @@ Please see the list below:
 |                  ternary operator                  |    :question:     |
 |                 indexed parameters                 |        :x:        |
 |                      receive                       |    :question:     |
+|         named arguments in function calls          |    :question:     |
+|                     Yul Blocks                     |    :question:     |
+|             reverts with custom errors             |    :question:     |
+|                user defined errors                 |    :question:     |
+|               function call options                |    :question:     |
+| member access of address object (address.balance)  |    :question:     |
```

Comment:
this should say "e.g. address.balance" to make it clear all the address members are disallowed